### PR TITLE
fix: TypeScript performance of `BarPercent.tsx`

### PR DESCRIPTION
**Pull request by bnussman-akamai**
## Description 📝

- Improves the TypeScript type-checking performance on the `BarPercent.tsx` component 🏎️ 

## Preview 📷

| Before  | After   |
| ------- | ------- |
| Screenshot 2024-10-09 at 1 58 53 PM | Screenshot 2024-10-09 at 1 54 17 PM |
| takes 16 seconds to typecheck | takes 14ms to type check 😦| 

## How to test 🧪

- Verify type checking passes in Github Actions
- Verify there are no regressions with the component, it's types, and it's behavior
- Profile this if you want to (`yarn tsc --generateTrace ./trace --incremental false` and inspect with `chrome://tracing`)

## As an Author I have considered 🤔

- [ ] 👀 Doing a self review
- [ ] ❔ Our [contribution guidelines](https://github.com/linode/manager/blob/develop/docs/CONTRIBUTING.md)
- [ ] 🤏 Splitting feature into small PRs
- [ ] ➕ Adding a [changeset](https://github.com/linode/manager/blob/develop/docs/CONTRIBUTING.md#writing-a-changeset)
- [ ] 🧪 Providing/Improving test coverage
- [ ] 🔐 Removing all sensitive information from the code and PR description
- [ ] 🚩 Using a feature flag to protect the release
- [ ] 👣 Providing comprehensive reproduction steps
- [ ] 📑 Providing or updating our documentation
- [ ] 🕛 Scheduling a pair reviewing session
- [ ] 📱 Providing mobile support
- [ ] ♿  Providing accessibility support

### CODE REVIEW

File: packages/manager/src/components/BarPercent/BarPercent.tsx [hunk 1]
```diff
@@ -1,10 +1,11 @@
 import { styled } from '@mui/material/styles';
-import { SxProps } from '@mui/system';
```
Comment (bnussman-akamai) [inline packages/manager/src/components/BarPercent/BarPercent.tsx:2]: This is another win for pnpm (https://github.com/linode/manager/pull/11000). By nature it won't allow imports from a package that are not an explicit dependency. 

Here, we are importing from `@mui/system` which is **not** defined in our `package.json`

Comment (abailly-akamai) [reply packages/manager/src/components/BarPercent/BarPercent.tsx:2]: > By nature it won't allow imports from a package that are not an explicit dependency.

music to my ears

**Review by abailly-akamai — APPROVED** [review]

File: packages/manager/src/components/BarPercent/BarPercent.tsx (hunk 1 above)
Comment (carrillo-erik) [reply packages/manager/src/components/BarPercent/BarPercent.tsx:2]: Should these changes be applied to other instances where we are importing `@mui/system` as well?

Screenshot 2024-10-09 at 12 17 02 PM

**Review by carrillo-erik — APPROVED** [review]

File: packages/manager/src/components/BarPercent/BarPercent.tsx (hunk 1 above)
Comment (bnussman-akamai) [reply packages/manager/src/components/BarPercent/BarPercent.tsx:2]: yeah, I believe so. I'll do a follow up PR for this!

**Pull request merged**